.github/workflows/pull.yml

```diff
@@ -5,36 +5,39 @@ on:
   pull_request:
 
 jobs:
+  pr-populate-dafny-versions:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Populate Dafny versions list
+      id: populate-dafny-versions-list
+      run: echo "dafny-versions-list=['4.1.0', '4.3.0']" >> $GITHUB_OUTPUT
+    outputs:
+      dafny-version-list: ${{ steps.populate-dafny-versions-list.outputs.dafny-versions-list }}
```

Comment:
Wow... GitHub needs to work on this...

Reply:
And good luck if you want to pass values between WORKFLOWS :)